pkg/daemon/daemon.go

```diff
@@ -822,6 +837,64 @@ func tryEnableRdma() (bool, error) {
 	return false, err
 }
 
+func tryCreateSwitchdevUdevRule(nodeState *sriovnetworkv1.SriovNetworkNodeState) error {
+	glog.V(2).Infof("tryCreateSwitchdevUdevRule()")
+	var new_content string
+	filePath := "/host/etc/udev/rules.d/20-switchdev.rules"
+
+	for _, ifaceStatus := range nodeState.Status.Interfaces {
+		if ifaceStatus.EswitchMode == sriovnetworkv1.ESWITCHMODE_SWITCHDEV {
+			switchID, err := utils.GetPhysSwitchID(ifaceStatus.Name)
+			if err != nil {
+				return err
+			}
+			portName, err := utils.GetPhysPortName(ifaceStatus.Name)
+			if err != nil {
+				return err
+			}
+			new_content = new_content + fmt.Sprintf("SUBSYSTEM==\"net\", ACTION==\"add|move\", ATTRS{phys_switch_id}==\"%s\", ATTR{phys_port_name}==\"pf%svf*\", IMPORT{program}=\"/etc/udev/switchdev-vf-link-name.sh $attr{phys_port_name}\", NAME=\"%s_$env{NUMBER}\"\n", switchID, strings.TrimPrefix(portName, "p"), ifaceStatus.Name)
+		}
```

Comment:
in os-net-config we have udev rule unmanage vf_representors by NM. I think you should added it as well.

https://github.com/openstack/os-net-config/blob/3144a8fc4fae95e3cfdc680ccdcbd5144a9a17a8/os_net_config/sriov_config.py#L369-L373

Reply:
Done